.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
+
+language: go
+go:
+  - 1.4.3
+  - 1.5.1
+  - tip
+
+install:
+  - go get -d ./...
+
+script:
+  - go test -v ./...
```

Comment:
There is no need to define `install` or `script`. Travis executes exactly these commands by default.